data/json/recipes/recipe_others.json

```diff
@@ -419,6 +419,36 @@
   ]
 },{
   "type" : "recipe",
+  "result": "gasoline_lantern",
+  "category": "CC_OTHER",
+  "subcategory": "CSC_OTHER_TOOLS",
+  "skill_used": "fabrication",
+  "difficulty": 5,
+  "time": 60000,
+  "reversible": false,
+  "autolearn": true,
+  "qualities":[
+    {"id":"HAMMER","level":1,"amount":1}
+  ], "components": [
+    [
+      [ "scrap", 1 ]
+    ],
+    [
+      [ "rag", 1 ]
+    ],
+    [
+      [ "bottle_glass", 1 ],
+      [ "flask_glass", 1],
+      [ "jar_glass", 1 ],
+      [ "clay_teapot", 1 ],
```

Comment:
Yeah, it kinda worked for oil lamps (KA101 thinks Aladdin-style lamps) but not here.  Fair enough.